amgut/handlers/open_humans.py

```diff
@@ -75,7 +75,7 @@ class OpenHumansLoginHandler(BaseHandler, OpenHumansMixin):
     _API_URL = urljoin(AMGUT_CONFIG.open_humans_base_url, '/api')
 
     _OAUTH_REDIRECT_URL = urljoin(AMGUT_CONFIG.base_url,
-                                  '/authed/connect/open-humans/')
+                                  'authed/connect/open-humans/')
```

Comment:
The overloaded `redirect` only works under this condition:

- the URL starts with a `/`
- the URL doesn't start with SITEBASE

In this case, `AMGUT_CONFIG.base_url` should be `https://microbio.me/AmericanGut/`, which doesn't start with a slash, so the overloaded redirect doesn't affect it (or any other absolute URL). :)